app/components/common/core/core.js

```diff
@@ -1,4 +1,4 @@
-(function() {
+// (function() {
```

Comment:
I think I was testing something.  I can uncomment it in my next pull request if ur cool with that.